### PR TITLE
Ea fix upgrade

### PR DESCRIPTION
**Pull request by chkp-yonatanph**

### CODE REVIEW

File: checkpoint/cp-resource-management/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: v1.2.1
-version: 1.08.7
+version: 1.09.0
```
Comment (chkp-rigor) [inline checkpoint/cp-resource-management/Chart.yaml:3]: I prefer to keep 1.08.* for ea branch, i.e. to change to 1.08.8

File: checkpoint/cp-resource-management/README.md [hunk 2]
```diff
@@ -16,21 +16,32 @@ Check Point ClougGuard provides Compliance, Vulnerability Assessment, Visibility
...
 ```
 
-This command deploys a CloudGuard Resource Management agent.
+This command deploys a CloudGuard Resource Management agent as well as optional add-ons.
+
 
 > **Tip**: List all releases using `helm list`
 
+
+## Upgradeing the chart
```
Comment (chkp-rigor) [inline checkpoint/cp-resource-management/README.md:36]: type (Upgrad**e**ing -->Upgrading)

File: checkpoint/cp-resource-management/README.md [hunk 3]
```diff
@@ -16,21 +16,32 @@ Check Point ClougGuard provides Compliance, Vulnerability Assessment, Visibility
...
+
+To upgrade the deployment and/or to add/remove additional feature run:
+
+```bash
+$ helm repo update
+$ helm upgrade asset-mgmt checkpoint/cp-resource-management --set addons.imageUploader.enable=[true/false] --set addons.flowLogs.enable=[true/false] --reuse-values --namespace=[Namespace]
```
Comment (chkp-rigor) [inline checkpoint/cp-resource-management/README.md:42]: remove `--reuse-values `

You should mention that all enabled charts (and in fact all flags with non-default values) should be set in the upgrade command.

File: checkpoint/cp-resource-management/README.md [hunk 4]
```diff
@@ -47,10 +58,10 @@ In order to get the [Check Point ClougGuard](https://secure.dome9.com/) Cluster
 
 Refer to [values.yaml](values.yaml) for the full run-down on defaults. These are a mixture of Kubernetes and CloudGuard directives that map to environment variables.
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+Specify each parameter using `--set key=value[,key=value]` or `--set-string key=value[,key=value]` to `helm install`. For example,
```
Comment (chkp-rigor) [inline checkpoint/cp-resource-management/README.md:61]: since it's not helm documentation, I would leave only the first syntax.

Comment (chkp-rigor) [reply checkpoint/cp-resource-management/README.md:61]: Do you want to mention it's relevant not only to `helm install` but to `helm upgrade `too?

**Review by chkp-rigor — COMMENTED** [review]
few minor comments

File: checkpoint/cp-resource-management/README.md [hunk 5]
```diff
@@ -16,21 +16,32 @@ Check Point ClougGuard provides Compliance, Vulnerability Assessment, Visibility
...
 
 ```bash
-$ helm repo add checkpoint https://raw.githubusercontent.com/CheckPointSW/charts/master/repository/
-$ helm install my-release checkpoint/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
+$ helm repo add checkpoint-ea https://raw.githubusercontent.com/CheckPointSW/charts/ea/repository/
+$ helm install asset-mgmt checkpoint-ea/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
 ```
 
-These are the additional optional flags for available add-ons:
+These are the additional optional flags for to enable add-ons:
```
[the remark this answers is not in the document]
Comment (chkp-yonatanph) [reply checkpoint/cp-resource-management/README.md:23]: hmmm could be either way.
It might be:
These are the flags to enable additional optional add-ons.

**Review by chkp-rigor — APPROVED** [review]
great!

**Pull request merged**